### DISCUSSION TITLE
Enabling HTTPS

### DISCUSSION THREAD

**Discussion by ea1fsc** [wavelog/wavelog#477] [Q&A] [answered]
Hi to all,
I would like to enable HTTPS to my Wavelog installation, but I haven't found anything about it in the documentation. I have tried to enable it like other Apache installation but it doesn't work. 
I am using Apache2 as my webserver.

**Comment by HB9HIL** [comment, marked as answer]
Hello @juantxi17 

Make sure you also set https in the URL which is set in your `application/config/config.php`

`$config['base_url']	= 'https://your_url';`

**Reply by ea1fsc** [reply]
Perfect, that worked! Thanks you so much.
